assets/src/scripts/wdfn-home/store/wdfn-store.js

```diff
@@ -191,11 +191,14 @@ export const retrieveWdfnData = function ({ siteTypes, bBox, parameters, timePer
           variables.startDateLo = timePeriod;
         }
 
+        dispatch(applyLoadingState('loading'));
```

Comment:
Setting this here rather than in the search component makes more sense so it can easily be re-set whenever a search comes back, whether or not there are changed values.